sample/src/main/kotlin/com/slack/circuit/sample/home/HomePresenter.kt

```diff
@@ -135,15 +136,16 @@ fun HomeContent(state: HomeScreen.State, eventSink: (HomeScreen.Event) -> Unit)
       BottomNavigationBar(selectedIndex = state.homeNavState.index) { index ->
         eventSink(HomeScreen.Event.HomeEvent(HomeNavScreen.Event.HomeNavEvent(index)))
       }
-    },
-    content = {
-      if (state.homeNavState.index == DOGS_SCREEN_INDEX) {
-        PetList(state.petListState) { event -> eventSink(HomeScreen.Event.PetListEvent(event)) }
-      } else if (state.homeNavState.index == ABOUT_SCREEN_INDEX) {
-        About()
+    }
+  ) {
+    if (state.homeNavState.index == DOGS_SCREEN_INDEX) {
+      PetList(modifier = Modifier.padding(it), state = state.petListState) { event ->
```

Comment:
hmm, how does this pattern work with `CircuitContent`? Is this an example of #39?

Reply:
Maybe? I think since we pass in `LazyColumn` and `Row` currently which both themselves expose a `modifier` param it hasn't been an issue but I can see us needing to expose `modifier` once we have a lot more custom Compose functions. So to the question `Do we need to expose modifier in Ui.Render?` I think we do.